frontend/app/utils/date-utils.ts

```diff
@@ -60,3 +61,10 @@ export function useMonths(locale: string, format: 'numeric' | '2-digit' | 'long'
 export function toLocaleDateString(date: Date, locale: string) {
   return date.toLocaleDateString(`${locale}-CA`, { year: 'numeric', month: 'long', day: 'numeric' });
 }
+
+export function getAgeFromDateString(dateOfBirth: string) {
+  const parseDateOfBirth = parse(dateOfBirth, 'yyyy-MM-dd', new Date());
+  invariant(isValid(parseDateOfBirth), `date of birth is invalid [${dateOfBirth}]`);
```

Comment:
Is this call to `isValid(..)` required here? I am looking at the docs for `date-fns` and don't see any mention of it. Does `parse(..)` **not** throw if you give it an invalid input?

Reply:
(parse)[https://date-fns.org/v3.6.0/docs/parse] dont throw.

> If parsing failed, Invalid Date will be returned. Invalid Date is a Date, whose time value is NaN. Time value of Date: http://es5.github.io/#x15.9.1.1